templates/base.html

```diff
@@ -4,229 +4,233 @@
 {% load render_bundle from webpack_loader %}
 
 <html lang="{{ LANGUAGE_CODE }}" class="no-js">
-    <head>
-        <title>{% block title %}{{ settings.name }}{% endblock %}</title>
-        {% block meta %}
+<head>
```

Comment:
You changed indent for 4 spaces in this file? Please check other files to be consistent.

Reply:
Yeah, you're right. It should be cool now.